### PR TITLE
Plugin removal

### PR DESCRIPTION
**Pull request by nablaone**
Plugins were premature.  We need to design a proper architecture first.

### CODE REVIEW

File: quesma/jsonprocessor/ingest.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
+// Copyright Quesma, licensed under the Elastic License 2.0.
```
Comment (avelanarius) [inline quesma/jsonprocessor/ingest.go:1]: The existing `quesma/jsonprocessor/json_processor.go` file is also used only for ingest, so maybe both of them should live in a single file.

However, I'm not sure if there should be a separate package`jsonprocessor (due to a risk of introducing circular dependencies later)

File: quesma/model/transformers.go [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// Copyright Quesma, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+package model
```
Comment (pdelewski) [inline quesma/model/transformers.go:3]: Not sure if that should be part of model, but we can revisit this later.

File: quesma/jsonprocessor/ingest.go (hunk 1 above)
Comment (pdelewski) [reply quesma/jsonprocessor/ingest.go:1]: +1

**Review by pdelewski — APPROVED** [review]

**Comment by jakozaur** [comment]
Let's talk about it during standup before merging.

**Review by jakozaur — APPROVED** [review]

**Pull request merged**